website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -545,8 +545,8 @@ variables is shown in the table below.
 | `orca_default_recovery_type`                          | string        | The Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited'                                               | "Standard" |
 | `orca_default_storage_class`                          | string        | The [class of storage](../../operator/storage-classes.md) to use when ingesting files. Can be overridden by collection config. | "GLACIER" |
 | `orca_delete_old_reconcile_jobs_frequency_cron`       | string        | Frequency cron for running the delete_old_reconcile_jobs lambda.                                                               | "cron(0 0 ? * SUN *)" |
-| `orca_ingest_lambda_memory_size`                      | number        | Amount of memory in MB the ORCA copy_to_glacier lambda can use at runtime.                                                     | 2240 |
-| `orca_ingest_lambda_timeout`                          | number        | Timeout in number of seconds for ORCA copy_to_glacier lambda.                                                                  | 600 |
+| `orca_ingest_lambda_memory_size`                      | number        | Amount of memory in MB the ORCA copy_to_archive lambda can use at runtime.                                                     | 2240 |
+| `orca_ingest_lambda_timeout`                          | number        | Timeout in number of seconds for ORCA copy_to_archivelambda.                                                                  | 600 |
```

Comment:
Need a space `copy_to_archivelambda` to `copy_to_archive lambda`

Reply:
Fixed